$NetBSD$

Adjust go version in order to match expected format with only major and
minor per Go 1.20.

--- go.work.orig	2023-10-03 20:22:19.000000000 +0000
+++ go.work
@@ -1,4 +1,4 @@
-go 1.21.1
+go 1.21
 
 use (
 	.
